controllers/index.js

```diff
@@ -0,0 +1,10 @@
+const { update } = require("lodash");
+const { selectTopics } = require("../models/index");
+exports.getTopics = (req, res, next) => {
+  console.log("in the controller");
```

Comment:
Always a good idea to remove console.logs as soon as you don't need them anymore to keep your code clean